assets/components/index.js

```diff
@@ -7,5 +7,35 @@ var timeoutDialog = require('../patterns/help-users-when-we-time-them-out-of-a-s
 ;(function ($, window, document) {
   accountMenu()
   charCounter()
-  $.timeoutDialog = timeoutDialog
+  window.govuk = window.govuk || {}
```

Comment:
Should we capitalise the namespace? I think there might already be a window.GOVUK though?